x-pack/platform/plugins/shared/observability_ai_assistant/server/service/client/operators/get_generated_title.ts

```diff
@@ -8,12 +8,14 @@
 import { catchError, mergeMap, Observable, of, tap, from } from 'rxjs';
 import { Logger } from '@kbn/logging';
 import { ChatCompleteResponse } from '@kbn/inference-common';
+import type { AssistantScope } from '@kbn/ai-assistant-common';
 import type { ObservabilityAIAssistantClient } from '..';
 import { Message, MessageRole } from '../../../../common';
 
 export const TITLE_CONVERSATION_FUNCTION_NAME = 'title_conversation';
 export const TITLE_SYSTEM_MESSAGE =
-  'You are a helpful assistant for Elastic Observability. Assume the following message is the start of a conversation between you and a user; give this conversation a title based on the content below. DO NOT UNDER ANY CIRCUMSTANCES wrap this title in single or double quotes. This title is shown in a list of conversations to the user, so title it for the user, not for you.';
+  'You are a helpful assistant for {scope}. Assume the following message is the start of a conversation between you and a user; give this conversation a title based on the content below. DO NOT UNDER ANY CIRCUMSTANCES wrap this title in single or double quotes. This title is shown in a list of conversations to the user, so title it for the user, not for you.';
+
```

Comment:
This feels brittle. What about this?

```suggestion
export const getTitleSystemMessage = (scope: string) =>
  `You are a helpful assistant for ${scope === 'observability' ? 'Elastic Observability' : 'Elasticsearch'}. Assume the following message is the start of a conversation between you and a user; give this conversation a title based on the content below. DO NOT UNDER ANY CIRCUMSTANCES wrap this title in single or double quotes. This title is shown in a list of conversations to the user, so title it for the user, not for you.`;
```